YDS-Storybook/SwiftUI/Resources/SwiftUIYDSTypoArray.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  SwiftUIYDSTypoArray.swift
+//  YDS-Storybook
+//
+//  Created by 박지윤 on 2023/08/22.
+//
+// swiftlint:disable line_length
+
+import SwiftUI
+import YDS_SwiftUI
+
+struct WrappedSwiftUITypo {
+    let font: Font?
+    let typoStyle: String
+    let size: String
+    let weight: String
+    let lineHeight: String
+}
+
+extension WrappedSwiftUITypo: CustomStringConvertible {
+    var description: String {
+        return self.typoStyle
+    }
+}
+
+extension WrappedSwiftUITypo: Identifiable {
+    var id: String {
+        return self.typoStyle
+    }
+}
+
+struct WrappedSwiftUITypos {
+    let items: [WrappedSwiftUITypo]
+    let description: String?
+}
+
+extension WrappedSwiftUITypos: Identifiable {
+    var id: UUID {
+        UUID()
+    }
+}
+
+struct SampleLabel {
+    static let sampleLabel = """
+        계절이 지나가는 하늘에는 가을로 가득 차 있습니다.
+        나는 아무 걱정도 없이 가을 속의 별들을 다 헬 듯합니다.
+        가슴 속에 하나 둘 새겨지는 별을 이제 다 못 헤는 것은 쉬이 아침이 오는 까닭이요, 내일 밤이 남은 까닭이요, 아직 나의 청춘이 다하지 않은 까닭입니다.
+        """
+}
```

Comment:
```suggestion
struct SampleText {
    static let text = """
        계절이 지나가는 하늘에는 가을로 가득 차 있습니다.
        나는 아무 걱정도 없이 가을 속의 별들을 다 헬 듯합니다.
        가슴 속에 하나 둘 새겨지는 별을 이제 다 못 헤는 것은 쉬이 아침이 오는 까닭이요, 내일 밤이 남은 까닭이요, 아직 나의 청춘이 다하지 않은 까닭입니다.
        """
}
```
SampleLabel 말고 다른 이름은 어떠신가요?

Reply:
반영했습니다~!